modAionImpl/src/org/aion/zero/impl/cli/Arguments.java

```diff
@@ -258,4 +266,8 @@ public String getDumpStateCount() {
     public boolean isDbCompact() {
         return dbCompact;
     }
+
+    public String isReImport() {
```

Comment:
nit: "isReimport"

Reply:
I went for refactoring to `--redo-import`